constants.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file contains constant values that are not expected to change throughout
+the run of a program
+"""
+WEATHER_FILES_DIR = "weatherfiles/"
+TEMPERATURE_UNIT = "C"
+HUMIDITY_UNIT = "%"
+first_line = open(f"{WEATHER_FILES_DIR}Murree_weather_2004_Aug.txt", "r").readline()
```

Comment:
I need more clarification with this comment. because we are parsing the cli parameters in weatherman.py. does this mean i should parse the cli parameters in constants.py?

Reply:
I need more clarification with this comment. because we are parsing the cli parameters in weatherman.py. does this mean i should parse the cli parameters in constants.py?

Reply:
I mean we are using `Murree_weather_2004_Aug.txt` file as a reference to getting the column names. What if this file doesn't exist on the provided path?

Reply:
Done